Add new center position

bastiengrignon opened the pull request.

bastiengrignon committed:
Adding center/middle center position for the notification

bastiengrignon committed:
Merge pull request #1 from bastiengrignon/feature/position-notifications-update

Adding new center position for the notification

salamander2 commented:
Warning: as far as I can tell, only the dist/ has been updated, the src/ has not.

bastiengrignon committed:
ADD new CSS positions

bastiengrignon committed:
ADD CCS positions as valid

bastiengrignon committed:
ADD position to the dropdown of the demo page

bastiengrignon committed:
UPDATE readme with new position

bastiengrignon commented:
@salamander2 Here is my code updated with new positions,  updated README and demo html file

salamander2 commented:
@bastiengrignon   Thanks! I haven't had time to do any stuff here, but I'll look at it in the next few days.

salamander2 commented:
@bastiengrignon   okay. looks really good. I haven't tested it yet though.  

I think that you should update the Readme.MD to say that you are taking over maintenance of this app (or whatever it's called) since JLivingstone is AWOL.  Just take it over. The original one points to you via the fork.  
I don't think that there is much else to do other then spruce up the readme a bit (like "build|unknown")

salamander2 committed:
Improving demo. add text of javascript command that is executed.
Attempted to color code the themes in the listbox.

salamander2 committed:
Fixed duration error in demo. Also added screenshot.

salamander2 committed:
update Readme.md with example image.

salamander2 committed:
Update demo/demo.js

Co-authored-by: Bastien Grignon <41306995+bastiengrignon@users.noreply.github.com>

salamander2 committed:
fixed up CSS for <select><option>

salamander2 committed:
removed TODO from readme.md

bastiengrignon committed:
Merge pull request #2 from salamander2/master

Improving demo. Add text of javascript command that is executed. Attempted to color code the themes in the listbox.

bastiengrignon committed:
Cleanup code with new javascript 

CONVERT function to arrow function
CONVERT old var keyword to new const or let
FIX missing semi-colon

bastiengrignon committed:
UPDATE badge for Travis CI
